packages/universal-components/src/RangeDatePicker/RangeDatePickerTypes.js

```diff
@@ -9,6 +9,8 @@ export type ConfirmType = {|
 |};
 
 export type Props = {|
+  +calendarStartDate?: Date,
+  +isChoosingPastDatesEnabled?: boolean,
```

Comment:
I'm slightly confused by this new functionality. Shouldn't this be handled just by setting `calendarStartDate`?
When I set start date to for example 10. 4. , I would expect that picker will be selectable from this date. Instead of it, the month is rendered, but still not selectable. 

<img width="355" alt="Screenshot 2019-05-30 at 14 59 02" src="https://user-images.githubusercontent.com/2660330/58634678-2bc90d80-82ec-11e9-9ffe-848e9df9cd4e.png">

And when I choose `isChoosingPastDatesEnabled` option, whole fourth month is selectable.

Same with the higher month. When I choose 10. 6. as start. Month is again selectable from start.

<img width="352" alt="Screenshot 2019-05-30 at 14 59 26" src="https://user-images.githubusercontent.com/2660330/58634684-32578500-82ec-11e9-9117-bc77aef10d95.png">

I'm not sure how important is this feature right now. We can update it to make exact `calendarStartDate` limit possible selection range. Then `isChoosingPastDatesEnabled` will not be necessary because I will be able to set start date exactly to day I want. As addition to it we can also have something like `calendarEndDate` to specify end limit for selection range (can replace `numberOfRenderedMonths` functionality).

O at least rename`calendarStartDate` to something like `calendarStartMonth` (or some object which will have month and year props) to be clear that actual day doesn't really matter.

Reply:
Good point. I didn't realise that this name could be confused with functionality of `isChoosingPastDatesEnabled`. It makes sense to rename it. 

Your other proposed solution is a bit more complicated and if we don't need it now (we don't), let's skip it for now.

Reply:
Ok, if this functionality is not crucial right now definitely just some renaming should be enough.
Just to be clear, I'm ok with the `isChoosingPastDatesEnabled`. But it should be somehow clear  that `calendarStartDate` doesn't really define exact starting date/day and only tells which month will be rendered as first one. Thanks

Reply:
Done